### ISSUE TITLE
A11y_A11y_.NETOnlineREPL_WCAG2.1_.NET Editor_Status Message:Screen reader does not announce the updated results in the output section after activating "Run" button.

### ISSUE THREAD

**Issue by Charanrajkb** [dotnet/try#788]
**Environment details:**
https://docs.microsoft.com/
Browser: Chromium Edge Version 80.0.361.50 (Official build) (64-bit)
OS Details:Windows 10 

GitHubTags:#A11yTCS;#A11ySev2;#A11y_.NETOnlineREPL_WCAG2.1;#A11yWCAG2.1;#.NET Online REPL;#Benchmark;#DesktopWeb;#Win10;#MAS4.1.3;#Accessibility;#.NET Online REPL;#ChromiumEdge;

**​Repro Steps:​**
1.Hit the URL "https://docs.microsoft.com/" and login with appropriate credentials.
2.Tab till "Enter focus Mode" button and press enter.
3.Start Narrator and tab till Run button and hit enter.
4.Observe the narrator reading.

**Actual Results:​**
Screen reader does not announce the updated status in the output section after activating "Run" button.
After activating the Run button the status gets updated in the output section which is not announced by the screen reader.

**Same issue with Chrome + NVDA .**

**Expected Result:​**
After activating the Run button the status gets updated in the output section which must be announced by the screen reader.

**User Impact:​**
Screen readers users will not be aware about the results updated in the output section.

**Attachment:**
[788_A11y_.NETOnlineREPL_.NET Editor_Status Message.pptx](https://github.com/dotnet/try/files/4207879/788_A11y_.NETOnlineREPL_.NET.Editor_Status.Message.pptx)

**Comment by LadyNaggaga** [comment]
@Charanrajkb  I have notified the docs team of this bug [here](https://github.com/MicrosoftDocs/feedback/issues/2530)

**Comment by fsteffi** [comment]
this issue is fixed 
@LadyNaggaga 
can you please close this bughttps://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world?tutorial-step=1&WT.mc_id=-blog-scottha
